superset-frontend/src/components/Datasource/DatasourceEditor.jsx

```diff
@@ -971,9 +974,21 @@ class DatasourceEditor extends PureComponent {
         <EditLockContainer>
           <span role="button" tabIndex={0} onClick={this.onChangeEditMode}>
             {this.state.isEditMode ? (
-              <Icons.LockUnlocked iconColor={theme.colors.grayscale.base} />
+              <Icons.UnlockOutlined
+                iconSize="xl"
+                iconColor={theme.colors.grayscale.base}
+                css={theme => ({
+                  margin: `auto ${theme.gridUnit}px auto 0`,
+                })}
```

Comment:
```suggestion
                css={(theme) => css`
                  margin: auto ${theme.gridUnit}px auto 0;
               `}
```

Can we do this everywhere for consistency with the rest of the codebase? Just import `css` in the file.